src/components/Button/Button.tsx

```diff
@@ -28,10 +28,11 @@ export const Button = ({
   iconLayout = "sx",
   size = "medium",
   ...props
-}: Props) => {
+}: Props,ref) => {
```

Comment:
I think explicit Props typing becomes superfluous at this point

Reply:
done 👍